lib/logflare/backends/adaptor/bigquery_adaptor/finch/bidirectional_stream.ex

```diff
@@ -0,0 +1,101 @@
+defmodule Grpc.Client.Adapters.Finch.BidirectionalStream do
+  alias GRPC.Client.Adapters.Finch.StreamState
+  alias Grpc.Client.Adapters.Finch.RequestUtils
+  use GenServer
+
+  @finch_instance_name Logflare.FinchBQStorage
```

Comment:
If we're hardcoding it here then the application.ex should refer to this value instead through a public func

Reply:
And can't We pass it throguh as an option instead? So that we can keep the grpc client adaptor free of any BQ logic